share/doc/homebrew/Interesting-Taps-&-Branches.md

```diff
@@ -36,7 +36,7 @@ Homebrew has the capability to add (and remove) multiple taps to your local inst
 
 *   [homebrew/x11](https://github.com/Homebrew/homebrew-x11): Formulae with hard X11 dependencies.
 
-`brew search` looks in these main taps and as well in [Homebrew/homebrew](https://github.com/Homebrew/homebrew). So don't worry about missing stuff. We will add other taps to the search as they become well maintained and popular.
+`brew search` looks in these main taps as well as in [Homebrew/core](https://github.com/Homebrew/homebrew-core). So don't worry about missing stuff. We will add other taps to the search as they become well maintained and popular.
```

Comment:
I gave all-fixed-width a shot (https://github.com/apjanke/brew/blob/5892c3b067c3c817ba488cf8687b52bb6a69841e/share/doc/homebrew/Interesting-Taps-%26-Branches.md) and it looks ugly and "tech-y" to me; harder to read. Let's exercise some aesthetic discretion and not do this.

Tech books I've read, like O'Reilly's and Microsoft's, often differentiate between markup for proper/technical names/identifiers and literal code snippets: code snippets are monospace like we do here, but identifiers used in prose are rendered in bold or demibold. (Think I've seen italics, too.) Looks prettier and IMHO easier to scan when reading sections of prose (because you don't have to switch back and forth between monospace and proportional in text runs). I think this is especially true for GitHub pages viewed on the web, where they go so far as to change the background color for the monospace code sections.

I don't know if demibold is doable here (and there's definitely no Markdown syntax for it).

Bold is a little loud, but doesn't look bad to me on Firefox or Safari on OS X. Did that [here](https://github.com/apjanke/brew/blob/2fac240e3e41054abdf44f250cc9d49e731e0b21/share/doc/homebrew/Interesting-Taps-%26-Branches.md). That doesn't look bad; I think I more dislike how wide the base font is in the first place. Think it's definitely better than monospaced, and better than no markup at all, because it's clear which of these things are Technical Names.

Thoughts?